script/testing/oltpbench/README.md

```diff
@@ -108,24 +120,27 @@ Let the base directory be `noisepage/script/testing/oltpbench`
   - End-to-end performance
   - Nightly
 
-  For more details, check the [config files](https://github.com/cmu-db/noisepage/blob/master/script/testing/oltpbench/README.md#config-files) section below for more details
+  For more details, check the [config files](https://github.com/cmu-db/noisepage/blob/master/script/testing/oltpbench#config-files) section below for more details
 - `run_oltpbench.py`: entry point to run the OLTP bench tests
 - `test_oltpbench.py`: defines the `TestOLTPBench` class which manage the lifecycle of a test suite
 - `test_case_oltp.py`: defines the `TestOLTPBench` class which manage the lifecycle of a test case
 - `utils.py`: defines a list of utility functions specifically used by OLTP bench tests
 
 ## Config files
-To run a OLTPBench test, you should run the `run_oltpbench.py --config-file=config.json`. 
+A config file is necessary to specify a test for OLTP Benchmark.
 
 ### Fields
 In the configuration file, those information are required:
 - A list of test cases in `testcases`
 - The benchmarks and options in each test case, required by the oltpbench's workload descriptor file
   - The `run_oltpbench` script will run all test cases in the configuration file sequentially. 
   - The `loop` key in the configuration file is used to duplicate the test case with different options.
-  - The `server_args` filed in the configuration specify the server command line args.
+  - The `server_args` object in the configuration specifies the command line arguments passed to the server when running the binary.
 
 ### Example:
+The following example shows a oltpbenchmark test with 2 testcases
```

Comment:
technically each item in the loop is a test case so it would be 1 TestSuite with 4 test cases

Reply:
fixed